### PR TITLE
Update installation doc with latest steps

### PR DESCRIPTION
**Pull request by lsm5**
Update installation doc with latest steps

- Remove Kubic repo suggestions where skopeo exists by default
- Include documentation about lack of Windows package
(RE: https://github.com/containers/skopeo/issues/715)

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@vrothberg @mtrmac @rhatdan @TomSweeneyRedHat @lbscorpio PTAL

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -62,19 +39,23 @@ sudo apk add skopeo
 brew install skopeo
 ```
 
+You may need to add registry certificate to your macOS truststore.
+```
+sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <registry_cert.crt>
```
Comment (mtrmac) [inline install.md:44]: This part doesn’t make sense to me.

- AFAICS this has nothing to do with installing Skopeo; just the `brew install` does that.
- WRT configuring a certificate trusted to connect to a registry, platform-agnostic paths like `--cert-dir` are still expected to work
- All of #563 ended up surprising to me. The original report was that that CAs from the macOS keychain were ignored, and the recommended fix is to … add the CA to the macOS keychain? Stated this way, it just does not follow at all, so there must be some subtlety to #563 that wasn’t written down.

Overall I think this part of the PR should just be dropped, and #563 either discussed further, or just closed as “2 years later, this was reported to work fine”.

**Review by mtrmac — COMMENTED** [review]
ACK other than the macOS keychain part.

Thanks!

File: install.md (hunk 1 above)
Comment (lsm5) [reply install.md:44]: ack, will do. thanks @mtrmac , and apologies if this bothered you on your PTO.

Comment (mtrmac) [reply install.md:44]: (No worries — I chose to take a little time to see what’s new, and I know I had the option not to. But I appreciate the thought all the same.)

**Comment by lsm5** [comment]
removed macos part. PTAL

**Comment by lsm5** [comment]
Also, added a note for Windows package absence. PTAL as well.

**Comment by lsm5** [comment]
@vrothberg @rhatdan @TomSweeneyRedHat tests are green. PTAL

**Comment by rhatdan** [comment]
LGTM

File: install.md [hunk 2]
```diff
@@ -97,17 +73,8 @@ sudo apt-get -y update
...
-Checkout the [Kubic project page](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/skopeo)
-for a list of supported Ubuntu version and
-architecture combinations. **NOTE:** The command `sudo apt-get -y upgrade`
-maybe required in some cases if Skopeo cannot be installed without it.
-The build sources for the Kubic packages can be found [here](https://gitlab.com/rhcontainerbot/skopeo/-/tree/debian/debian).
-
-CAUTION: On Ubuntu 20.10 and newer, we highly recommend you use Buildah, Podman and Skopeo ONLY from EITHER the Kubic repo
-OR the official Ubuntu repos. Mixing and matching may lead to unpredictable situations including installation conflicts.
+The [Kubic project](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/skopeo)
+provides packages for active Ubuntu 20.04 (it should also work with direct derivatives like Pop!\_OS).
```
Comment (vrothberg) [inline install.md:77]:
```suggestion
provides packages for Ubuntu 20.04 (it should also work with direct derivatives like Pop!\_OS).
```

Not sure what "active" means.

**Review by vrothberg — COMMENTED** [review]
I think that Ubuntu should get a dedicated section along with the info that skopeo is the main repositories starting with 20.10

File: install.md (hunk 2 above)
Comment (TomSweeneyRedHat) [reply install.md:77]: is it just 20.24 or "20.24 and later"?

Comment (lsm5) [reply install.md:77]: > Not sure what "active" means.

@vrothberg ha, `active` was a poor choice, I wanted to mean something that's not EOL, and yes, we could just get rid of that word.



> is it just 20.24 or "20.24 and later"?

@TomSweeneyRedHat in the case of podman, we're not mentioning kubic repos if the package exists in the default distro repos. I'm hoping to do the same here.

**Comment by lsm5** [comment]
> I think that Ubuntu should get a dedicated section along with the info that skopeo is the main repositories starting with 20.10

Ubuntu already has a dedicated section. Listed below debian and raspberry pi os. Can you check again and let me know if you meant something else?

**Review by vrothberg — COMMENTED** [review]
LGTM, thanks!

**Comment by vrothberg** [comment]
> > I think that Ubuntu should get a dedicated section along with the info that skopeo is the main repositories starting with 20.10
> 
> Ubuntu already has a dedicated section. Listed below debian and raspberry pi os. Can you check again and let me know if you meant something else?

Apologies, it looked like it was in the Debian section.

**Pull request merged**